### PR TITLE
fix(TRanimeizle): change metadata url

### LINKED ISSUE #7162: "TRanimeizle" service needs an update.
### Steps to Reproduce

Website's url changed, so it needs an update. New url: "tranimeizle.co"

### Expected Behavior

The app should show my activity

### Actual Behavior

It doesn't show my activity

### Page URL

tranimeizle.co

### Screenshots

_No response_

### Operating System

windows 7, 8, 10, 11

### Browser and Version

Brave Browser, latest version

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by RisingSunLight42**
## Description 

Resolves #7162 by changing the url

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

tranime_proof_1

tranime_proof_2

</details>

### CODE REVIEW

File: websites/T/TRanimeizle/metadata.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
...
 	},
 	"url": [
-		"www.tranimeizle.net",
-		"tranimeizle.net"
+		"www.tranimeizle.co",
+		"tranimeizle.co"
 	],
-	"version": "1.0.11",
+	"version": "1.0.12",
 	"logo": "https://i.imgur.com/TsLirN4.jpg",
```
Comment (DarkVillager) [inline websites/T/TRanimeizle/metadata.json:18]: Higher quality updated logo. (Also change in presence.ts please.)
```suggestion
	"logo": "https://i.imgur.com/BnMRZ49.png",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/T/TRanimeizle/metadata.json (hunk 1 above)
Comment (RisingSunLight42) [reply websites/T/TRanimeizle/metadata.json:18]: I'll do asap

**Review by DarkVillager — APPROVED** [review]

**Comment by RisingSunLight42** [comment]
if a verifier can rerun Presence Validator, I'll be grateful !

**Comment by tunahann0** [comment]
> if a verifier can rerun Presence Validator, I'll be grateful !

me too, i hope someone comes soon.

**Comment by DarkVillager** [comment]
> if a verifier can rerun Presence Validator, I'll be grateful !

or rebase :)

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**